### DISCUSSION TITLE
Released big update

### DISCUSSION THREAD

**Discussion by paullizer** [microsoft/AzureOpenAI-with-APIM#4] [Announcements]
Big updates this week!

1. Details on APIM Policies to support
   1. Monitoring and Chargeback
   2. Resiliency
   3. Scalability
   4. Performance
   5. Access Control
2. Updates to One-button Deploy
   1. Now supports Managed Identity for APIM to Azure OpenAI
   2. Supports Multi-region Azure OpenAI
   3. Supports retry policy
   4. Deploying Event Hub (will support Token count and Charge Back)
3. Documentation on Rate Throttling
4. Updated Architecture PNG
5. Added Architecture Visio
6. Consolidated Solution as Azure Government now supports deployment of Azure OpenAI
7. Updated Query Examples to use latest API

More Updates Coming for 
1. Specific guides on modifying existing APIM policy and services
   1. Monitoring and Chargeback
   2. Resiliency
   3. Scalability
   4. Performance
   5. Cost Management
   6. Cost Forecasting
2. Video Guidance on each component from deployment to modification of APIM
3. Load Balancing
4. Provisioned Throughput Units
5. Guidance on Managing Scale, Performance, Resiliency

**Comment by stateofthearb** [comment]
Hi Paul, Thanks for the awesome repo, just checking in to see if there is an eta on the chargeback announcement?
